### ISSUE TITLE
preedit is not updated after commit-unhandled command.

### ISSUE THREAD

**Issue by wengxt** [ueno/libskk#71]
This is basically because update_preedit is only called if the key is filtered. But as for commit-unhandled, the key is filtered or not is controlled by egg_like_newline.

This would leave preedit to be non-empty after press "Shift+A" Enter.

**Comment by lo48576** [comment]
I've overlooked this but fixed by #96?

**Comment by wengxt** [comment]
Yeah,  I think it is the case. I forgot I report this 6 years ago .
